omero/developers/whatsnew.txt

```diff
@@ -14,11 +14,16 @@ Breaking changes
 OMERO.web
 ^^^^^^^^^
 
-OMERO.web has undergone a major upgrade, updating jsTree to version 3 and
+OMERO.web has undergone a major upgrade, updating jsTree to version 3.0.8 and
 using json for all tree loading to substantially improve performance.
 
-See the :pythondoc:`OMERO.web python docs <omeroweb/omeroweb.html>` for more
-information.
+If you have web code that directly works with the jsTree, you'll need to
+update it according to the `jsTree docs <https://www.jstree.com/>`.
+
+A number of new urls are available in webclient, with the /api/ prefix that
```

Comment:
"URLs" please. Otherwise, once that and the "you'll" are fixed, this is good to merge thanks.